spec/features/basic_user_pending_request_spec.rb

```diff
@@ -4,30 +4,15 @@
 
 RSpec.feature 'Publish Record', type: :feature do
   before do
-    stub_request(:get, 'https://country.beta.openregister.org/record/zz.tsv')
-      .with(headers: { 'Accept' => '*/*', 'Accept-Encoding' => 'gzip, deflate', 'Host' => 'country.beta.openregister.org' })
+    stub_request(:get, 'https://country.register.gov.uk/record/zz.tsv')
```

Comment:
I know it's a stub but this should probably stay as `country.beta.openregister.org` for the same reason as above.